src/main/kotlin/frc/team5499/frc2018Kotlin/subsystems/Intake.kt

```diff
@@ -0,0 +1,16 @@
+package frc.team5499.frc2018Kotlin.subsystems
+
+object Intake : Subsystem() {
+
+    override fun update() {
+    }
+
+    override fun stop() {
+    }
+
+    override fun reset() {
+    }
+
+    override fun broadcastToDashboard() {
```

Comment:
Lowkey should just delete all the broadcastToDashboard functions

Reply:
ok